tools/training-aev-benchmark.py

```diff
@@ -249,20 +285,43 @@ def wrapper(*args, **kwargs):
         print('   {}'.format(torch.cuda.get_device_properties(i)))
         checkgpu(i)
 
-    print("\n\n=> Test 1: USE cuda extension, Energy training")
+    # Warming UP
+    if len(dataset_shuffled) < 100:
+        runcounter = -1
+        parser.runname = 'Warning UP'
+        print(f"\n\n=> Test 0: {parser.runname}")
+        torch.cuda.empty_cache()
+        gc.collect()
```

Comment:
If you are not lack of memory, please don't do this

Reply:
this is for record relatively accurate gpu memory usage, you think this is not necessary?

Reply:
If you record accurate gpu memory usage, you might not be able to record accurate speed.